### ISSUE TITLE
Bad performance on Genshin when launched from XXMI launcher

### ISSUE THREAD

**Issue by ZeLink3123** [SpectrumQT/XXMI-Launcher#5]
I'm not entirely sure how to forma this properly since this is my first time writing to report an issue but Genshin when launched from the launcher has some serious performance hits. This is with no mods loaded and nvidia filters on. Launching the game normally (with fps unlocker) gives me my normal performance, 120 absolute max settings. Another thing I've noticed is that the built in nvidia statistics feature cannot determine the framerate of the game when launched through the launcher

Edit: It seems like star rail has a similar issue with performance and wuwa has a related issue. In wuwa's case, nvidia can't seem to detect it. In star rail's case both the filters and statistics feature do not work

Edit 2: It seems to be related to the updated dlls of the MIs as launching from an MI that i pathed to required me to update the dll, which led to this issue when launched from the same dll but not from the launcher

**Comment by SpectrumQT** [comment]
What's your **d3dx.ini**, what's your OS (+version build), what's your hardware? Were tests made with empty mods and shaderfixes folders? I highly doubt that it's DLL's fault, unless you're on really ancient OS or hardware.

**Comment by ZeLink3123** [comment]
Ok so Im on Windows 11 with a 5600x and 3060ti. All I really did was launch the game with the XXMI launcher. The MIs for each game were downloaded through the launcher itself, so the d3dx.ini should be the latest version.
However, I havent had issues with performance eversince I moved the MIs themselves into the XXMI launchers folder. Thats is to say I downloaded the MIs manually from github and moved them into the launcher folder. Everything runs fine if I do it that way
